src/schemas/space.json

```diff
@@ -324,9 +324,6 @@
               "type": "string",
               "enum": ["rejection"]
             },
-            "blind": {
```

Comment:
How is this related ?

Reply:
Looks like we are not using this field anymore. i assume we were using this field long ago to check if it shutter or not (need to remove on hub as well)

Reply:
maybe should i create a seperate PR?

Reply:
> maybe should i create a seperate PR?

Better, yes